### PR TITLE
Document Demo, reflow website to fit in demo

### PR DESCRIPTION
**Pull request by Laremere**
This adds the documentation for running the Open Match demo.  The overall flow is:

- Create cluster
- Install OM
- Get some basics on how the Demo and OM work.
- Install demo.
- View demo running, and read demo component source.
- Then start reading on a full understanding of OM.

### CODE REVIEW

File: site/content/en/docs/Getting Started/first_match.md [hunk 1]
```diff
@@ -1,20 +1,92 @@
...
-<TODO: This section is still work in progress>
+# Concepts for first match
 
-## Objectives
+A rough description of some important concepts.  Concrete examples are
+included in linked code samples in demo components.
 
-- Deploy the demo matchmaking function (MMF) and evaluator.
-- Create your first match.
+- **Ticket**: A ticket is a player (or group of players) requesting a match.
```
Comment (sawagh) [inline site/content/en/docs/Getting Started/first_match.md]: minor nit: but can we make this 'represents a ..' ?

File: site/content/en/docs/Getting Started/om_basics.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Understand Open Match"
 linkTitle: "Basic Concepts"
```
Comment (sawagh) [inline site/content/en/docs/Getting Started/om_basics.md]: Rename this to 'core concepts'?

**Comment by sawagh** [comment]
Can you add a note / link from the Contributing section back to the Development md file in the git repo?

File: site/content/en/docs/Getting Started/om_basics.md (hunk 2 above)
Comment (Laremere) [reply site/content/en/docs/Getting Started/om_basics.md]: https://github.com/googleforgames/open-match-docs/issues/22

**Pull request merged**